x-pack/plugin/relevance-search/src/main/java/org/elasticsearch/xpack/relevancesearch/relevance/curations/CurationsService.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.relevancesearch.relevance.curations;
+
+import org.elasticsearch.client.internal.Client;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Manage relevance settings, retrieving and updating the corresponding documents in .ent-search index
+ */
+public class CurationsService {
+
+    public static final String ENT_SEARCH_INDEX = ".ent-search";
+    public static final String CURATIONS_SETTINGS_PREFIX = "curations-";
+    private static final String CONDITIONS_FIELD = "conditions";
+    private static final String PINNED_DOCS_FIELD = "pinned_document_ids";
+    private static final String EXCLUDED_DOCS_IDS = "excluded_document_ids";
```

Comment:
Why do we switch terminology between hidden documents and excluded documents? Is there a distinction that I am not aware of?

If there isn't a distinction, we might want to consider choosing one and being consistent. 

Also, nitpick: I believe we should replace `IDS` with `FIELD` to be consistent with other field definition names.